e2e-tests/utils/assets.ts

```diff
@@ -0,0 +1,468 @@
+import { Page, expect } from "@playwright/test"
+import WalletPageHelper from "./walletPageHelper"
+
+export default class AssetsHelper {
+  constructor(
+    public readonly popup: Page,
+    public readonly walletPageHelper: WalletPageHelper
+  ) {}
+
+  /**
+   * This function verifies that the provided asset is present (an in case of
+   * base assets, displayed as first) on the list of assets on the Wallet page.
+   * It verifies that the asset has a balance (and a USD value, when aplicable)
+   * displayed and there are options to Send and Swap. It also makes sure the
+   * asset is not displayed in the unverified section.
+   */
+  async assertVerifiedAssetOnWalletPage(
+    assetSymbol: RegExp,
+    assetType: "base" | "knownERC20" | "trusted"
+  ): Promise<void> {
+    let asset: ReturnType<typeof this.popup.locator> | undefined
+    if (assetType === "base") {
+      asset = this.popup.getByTestId("asset_list_item").first() // We use `.first()` because the base asset should be first on the list
+    } else {
+      asset = this.popup.getByTestId("asset_list_item").filter({
+        has: this.popup.locator("span").filter({ hasText: assetSymbol }),
+      })
+    }
+    await this.popup.waitForTimeout(2000) // We need to wait for the background elements to disappear
+    await expect(asset.getByText(assetSymbol)).toBeVisible()
```

Comment:
We can set the timeout on the assertion itself too (unless for some reason it does not work)
```suggestion
    await this.popup.waitForTimeout(2000) // We need to wait for the background elements to disappear
    await expect(asset.getByText(assetSymbol)).toBeVisible({ timeout: 2000 })
```

Reply:
It seems that some of the recent changes made the timeout completely obsolete. I removed it in c21299e8369701e116968a3b17fc57d4ab626e62.